docs/src/componentDocs/Header/markdown/HeaderAPIDocs.mdx

```diff
@@ -40,7 +40,7 @@ The `<Header>` component is used at the top of the page to display page informat
 
 | Prop Name           | Description                                                                                                                | Type                                                                             | Required | Default                           |
 | ------------------- | -------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------| -------- | --------------------------------- |
-| actionItems         | Array of icons / actions to display on the right                                                                           | `Array<HeaderIcon \| HeaderActionComponent>`                                     | no       |                                   |
+| actionItems         | Array of icons / actions to display on the right                                                                           | `Array` [`HeaderIcon`](https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L34) \| [`HeaderActionComponent`](https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L42)                                  | no       |                                   |
```

Comment:
This is looking like:
<img width="854" alt="Screenshot 2024-05-22 at 4 27 34 PM" src="https://github.com/etn-ccis/blui-react-native/assets/120575281/77b5f09d-5543-4e9d-8238-d6a71559c075">

Should be updated to:

`Array` <[`HeaderIcon`](https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L34) \| [`HeaderActionComponent`](https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L42)>